README.md

```diff
@@ -16,6 +16,8 @@ This showcases are build around the Bitmovin Adaptive Streaming Player, demonstr
     * [ON_DOWNLOAD_FINISHED](events/onDownloadFinished.html): Report download errors for live streams using the ON_DOWNLOAD_FINISHED player event.
     * [eventConstants](events/eventConstants.html): Use event constants for registering event handlers at the player.
     * [ON_SEGMENT_REQUEST_FINISHED](events/onSegmentRequestFinished.html): Use ON_SEGMENT_REQUEST_FINISHED event to analyze which segment has been downloaded and implement custom workflows according to the HTTP status code.
+* [**frameaccurate**](frameaccurate/)
+    * [Frame Accurate Control](frameaccurate/js/FrameAccurateControls.ts): Bitmoinv Player Wrapper for SMTPE timestamp seeking + stepping
```

Comment:
We usually link the demos here

Reply:
The other links i've seen link to demos which consist of a single html file, i thought the SMTPE Controller would be more interesting here

Reply:
Makes sense, but I think the entry point of a demo is still the best option, to see which files are linked and so on. It's easy to navigate to all the included JS files, but it's much harder to go the other way around.